src/rendering/renderable.cpp

```diff
@@ -220,7 +249,9 @@ void Renderable::onEnabledChange(std::function<void(bool)> callback) {
 }
 
 void Renderable::setRenderBinFromOpacity() {
-    if (_renderBin != Renderable::RenderBin::PostDeferredTransparent) {
+    if ((_renderBin != Renderable::RenderBin::PostDeferredTransparent) &&
+        (_renderBin != Renderable::RenderBin::Overlay))
```

Comment:
What's the motivation for adding the `Overlay` here? (Wondering why it wasn't there before)

Also, with this change, this code is exactly the same as the onChange being registered in `registerUpdateRenderBinFromOpacity`, so that onChange should probably just call this function.